library/src/scala/tasty/Reflection.scala

```diff
@@ -1348,9 +1348,22 @@ trait Reflection extends reflect.Types { reflectSelf: CompilerInterface =>
 
   object Type:
 
+    /** Get the Type representing the given class */
     def apply(clazz: Class[_])(using ctx: Context): Type =
       reflectSelf.Type_apply(clazz)
 
+    /** Get package type if package is either defined in current compilation run or present on classpath. */
+    def staticPackageRef(path: String)(using ctx: Context): Type = reflectSelf.Type_staticPackageRef(path)
+
+    /** Get class type if class is either defined in current compilation run or present on classpath. */
+    def staticClassRef(path: String)(using ctx: Context): Type = reflectSelf.Type_staticClassRef(path)
```

Comment:
For non-package types, a safer alternative to passing a String is to pass a ClassTag (although that only works for things on the compilation classpath), this is how requiredClass works in scalac: https://github.com/scala/scala/blob/9c30f549886f93b3bb15dfdcebcfb47ef4f6a5fa/src/reflect/scala/reflect/internal/Definitions.scala#L349

Reply:
We already have that one with `Type.apply` just above. This one is for when you do not have the class available on the classpath.

Reply:
I see, since we already have a safe alternative, wouldn't it be better for the unsafe alternative to return an Option?

Reply:
These report an error as in dotty itself. They expect the definitions to be present in the classpath.

Reply:
But what type gets returned to the user?